server.go

```diff
@@ -32,39 +36,126 @@ const (
 	runtime        = "Microsoft AzureKMS"
 	runtimeVersion = "0.0.1"
 	configFilePath  = "/etc/kubernetes/azure.json"
+	azureResourceURL = "https://management.azure.com/"
+	vaultResourceURL = "https://vault.azure.net"
```

Comment:
sovereign clouds will carry a different url

Reply:
what's the best place to store this?